### PR TITLE
Square.OkIO 2.5.0

### PR DESCRIPTION
**Pull request by antonioseric**

### CODE REVIEW

File: Android/SquareOkIO/samples/OkIOSample/OkIOSample.csproj [hunk 1]
```diff
@@ -47,8 +47,8 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Android.Support.v7.AppCompat" Version="28.0.0.1" />
-    <PackageReference Include="Xamarin.Kotlin.StdLib" Version="1.3.61" PrivateAssets="None" />
+    <PackageReference Include="Xamarin.Android.Support.v7.AppCompat" Version="28.0.0.3" />
```
Comment (moljac) [inline Android/SquareOkIO/samples/OkIOSample/OkIOSample.csproj:50]: No need for AndroidX yet?

**Review by moljac — APPROVED** [review]
Thanks for contributions
Fala

File: Android/SquareOkIO/samples/OkIOSample/OkIOSample.csproj (hunk 1 above)
Comment (antonioseric) [reply Android/SquareOkIO/samples/OkIOSample/OkIOSample.csproj:50]: This is sample app..

Comment (moljac) [reply Android/SquareOkIO/samples/OkIOSample/OkIOSample.csproj:50]: Stupid me. Too much on my shouldeers...

**Pull request merged**